lib/parser/livesplit_core_version

```diff
@@ -1 +1 @@
-v0.10.0
\ No newline at end of file
+v0.11.0
```

Comment:
Since I needed LSC to be in the `Parser::` namespace, I took this opportunity to just update it as well.